### ISSUE TITLE
Web component: Mega Menu - "The essentials,  Hybrid Cloud" are read as menu options for screen reader users.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5081]
## Detailed description

On JAWS FF
Issue:
Screen reader is not supposed to identify the mentioned texts (The essentials,  Hybrid Cloud) as "menus". And if they are headings visually, They have to be identified as "heading" instead.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/861e5d47-ab9f-4ef0-bbd4-b86239e5a85d)

**Comment by annawen1** [comment]
@Praveen-sr can this be tested again? I'm tabbing through the menu and JAWS doesn't identify the "The essentials" or "Hybrid Cloud" text, should I be using some other keyboard command?

https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-masthead--with-custom-data

**Comment by Praveen-sr** [comment]
@annawen1 Re tested this on web component, issue still exists 
Steps to reproduce
1. After expanding "Product and solutions"
2. Use tab navigation to come to "what is hybrid cloud" link, then use up arrow keys to navigate through the text
"The essentials", "Hybrid Cloud".
Here "The essentials, Hybrid Cloud" are read as "menu" options for screen reader users.

**Comment by annawen1** [comment]
@Praveen-sr Ah ok, thank you for the detailed steps - will try that

**Comment by annawen1** [comment]
@Praveen-sr I tried the steps you mentioned, tabbing to the "what is hybrid cloud" link then using the arrows keys to navigate.

I am using a Virtual Machine and am trying to figure out how to enable audio but for now I have JAWS subtitles/captioning enabled and I do not see it mentioning "menu" when I navigate to the "The essentials" or "Hybrid cloud" text. I've posted a video. Please let me know if there is anything I'm missing or I should try.

https://user-images.githubusercontent.com/54281166/107439902-cbe6ba80-6b00-11eb-9e3d-65ed5205646c.mov

**Comment by annawen1** [comment]
@Praveen-sr I tried with JAWS 2020 version on both Firefox and Chrome but still can't replicate the issue

https://user-images.githubusercontent.com/54281166/107566398-d8c3e680-6bb2-11eb-9901-1a710625a081.mov

**Comment by Praveen-sr** [comment]
@annawen1 Hi, Ram was not able to screen record JAWS audio, instead he is provided the JAWS speech history, 

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/8c1ce785-0928-4719-85bd-83b51d09e0fe)
